From: Stewart Hildebrand <stewart.hildebrand@dornerworks.com>
Date: Fri, 13 Sep 2019 14:26:05 -0400
Subject: [PATCH] xen/arm: platform: additional Raspberry Pi compatible string

Upstream Linux kernel will use "brcm,bcm2711" as the compatible string
for Raspberry Pi 4 [1]. Add this string to our platform compatible list
for compatibility with the upstream kernel.

[1] https://patchwork.kernel.org/patch/11092621/

Signed-off-by: Stewart Hildebrand <stewart.hildebrand@dornerworks.com>
---
 xen/arch/arm/platforms/brcm-raspberry-pi.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xen/arch/arm/platforms/brcm-raspberry-pi.c b/xen/arch/arm/platforms/brcm-raspberry-pi.c
index e22d2b3184..a95a3db83f 100644
--- a/xen/arch/arm/platforms/brcm-raspberry-pi.c
+++ b/xen/arch/arm/platforms/brcm-raspberry-pi.c
@@ -21,6 +21,7 @@
 
 static const char *const brcm_bcm2838_dt_compat[] __initconst =
 {
+    "brcm,bcm2711",
     "brcm,bcm2838",
     NULL
 };
-- 
2.23.0

